Dockerfile

```
@@ -0,0 +1,7 @@
+FROM openjdk:11
+WORKDIR /
+ADD target/*.jar app.jar
```

```suggestion
COPY target/*.jar app.jar
```
[Best practice](https://docs.docker.com/develop/develop-images/dockerfile_best-practices/#add-or-copy)

---

done